internal/agent/device/applications/manager_test.go

```diff
@@ -61,31 +59,6 @@ func TestManager(t *testing.T) {
 			},
 			wantAppNames: []string{"app-new"},
 		},
-		{
```

Comment:
Note: moved to a separate test as this now requires an extra step to start the Monitor before stopping it.